UnitTests/Common/containers/lookuptable.drg

```diff
@@ -0,0 +1,73 @@
+Dragon library
+
+<header>
+[version]
+1.0.0
+
+[number of points]
+12
```

Comment:
Why is this together with the code?

Reply:
This seemed to me the best location to place the file because the unit tests are located in this folder. The only difference is that my unit test needs to read a file.

Reply:
Ok I thought the changes to the script were to allow you to use something larger from testcases.

Reply:
Now I get it, its for cloning the code, because otherwise theres only binaries.
This is fine.